octobot_backtesting/collectors/exchanges/exchange_collector.pxd

```diff
@@ -26,8 +26,19 @@ cdef class ExchangeDataCollector(data_collector.DataCollector):
     cdef public object tentacles_setup_config
     cdef public object start_timestamp
     cdef public object end_timestamp
+    cdef public bint finished
+    cdef public bint in_progress
+    cdef public int current_time_frame_index
+    cdef public int total_time_frame
+    cdef public int current_time_frame_percent
 
     cdef bint use_all_available_timeframes
 
+    cdef bint is_finished(self)
+    cdef bint is_in_progress(self)
+    cdef int get_current_time_frame_index(self)
+    cdef int get_total_time_frame(self)
+    cdef int get_current_time_frame_percent(self)
```

Comment:
`cdef` is for private methods, aren't those public (`cpdef`) ?

Reply:
ah yes ! sorry